burner-wallet-plugin/tokenbridge-bw-exchange/src/utils/utils.ts

```diff
@@ -7,7 +7,9 @@ export const constants = {
   EXCHANGE_TIMEOUT: 300000,
   MAX_FEE: toWei('1', 'ether'),
   ESTIMATE_FEE_MESSAGE: 'Estimation takes fee charges into consideration.',
-  ZERO_ADDRESS: '0x0000000000000000000000000000000000000000'
+  ZERO_ADDRESS: '0x0000000000000000000000000000000000000000',
+  HOME_TO_FOREIGN_FEE_TYPE: '0x741ede137d0537e88e0ea0ff25b1f22d837903dbbee8980b4a06e8523247ee26',
```

Comment:
Could you add in the comment a reference to the the corresponding tokenbridge contract?

Reply:
Sure